src/app/layout.tsx

```diff
@@ -28,7 +23,7 @@ export default function RootLayout({
   children: ReactNode
 }): ReactElement {
   return (
-    <html lang="en" className={`scroll-smooth ${font.className}`}>
+    <html lang="en" className="scroll-smooth">
```

Comment:
I suppose we still need `<OldFonts/>` applied to the rest of the pages like the blog

Reply:
The blog is in Pages router / Nextra 3, so it still has the font in `_app.tsx`.

Uri told me I can spend a moment refactoring this and migrating libraries, but... I just don't wanna do it yet. Let's get this conf website up, and then we can think about bumping Tailwind and unifying the routers, and doing whatever else

I kinda aligned on @kamilkisiela's opinion on SSG metaframeworks after reading some newest changelogs.